_posts/2022-11-30-amsterdam-bitdevs-is-back.md

```diff
@@ -23,5 +23,31 @@ within the event.
 
 # Agenda
 
-We will post the agenda a few days before the event, so be sure to check back!
-
+## Bitcoin v24.0 Release walkthrough
+
+* New `-mempoolfullrbf` flag
+  * Background
+  * Discussion
+* Anti-DoS headers sync: [#25717](https://github.com/bitcoin/bitcoin/pull/25717)
+* Deprecated RPCs
+* New RPCs
+  * `sendall`
+  * `simulaterawtransaction`
+* Wallet changes
+  * `-walletrbf` now default true
+  * Miniscript support
+  * Taproot multisig support
+  * Experimental `migratewallet` RPC
+
+## BIPs and Proposals
+* [BIP324](https://github.com/bitcoin/bips/pull/1378) - v2 P2P transport protocol
+
+## Research
+* Merkalize All The Things (MATT) - covenant proposal
```

Comment:
```suggestion
* Merkleize All The Things (MATT) - covenant proposal
```

Reply:
sigh, where were you 5 hours ago?

Reply:
In the Bitcoin Core PR Review Club :nerd_face: